frontend/components/Home.jsx

```diff
@@ -25,16 +26,16 @@ import "./plasmic/canvas_app_explorer/PlasmicHome.css"; // plasmic-import: 4XPgs
 export const Home__VariantProps = new Array();
 export const Home__ArgProps = new Array();
 function Home__RenderFunc(props,ref) {  
-    const [addedTools, setAddedTools] = useState([]); // each tool has one entry in array
+    //const [addedTools, setAddedTools] = useState([]); // each tool has one entry in array, for add/remove
     const [learnMoreActive, setLearnMoreActive] = useState([]); // each tool has one entry in array
     const [tools, setTools] = useState(null);
 
     useEffect(async () => {
-        const url = "/api/lti_tools/";
+        const url = "/api/lti_tools/"; // should be http://localhost:5000//api/lti_tools
```

Comment:
This is a relative URL. It will work and resolve to whatever domain it's running on. The full URL isn't needed here. In development it will be localhost but for prod and for testing LTI it will be running on a different domain.